i3pystatus/github.py

```diff
@@ -34,6 +35,7 @@ class Github(IntervalModule):
         ('unread_marker', 'sets the string that the "unread" formatter shows when there are pending notifications'),
         ("username", ""),
         ("password", ""),
+        ("access_token", ""),
```

Comment:
Can you put the link to the docs you mentioned here? Looks otherwise good, thanks.